lib/posix/timer.c

```diff
@@ -50,6 +50,7 @@ static void zephyr_timer_wrapper(struct k_timer *ztimer)
 int timer_create(clockid_t clockid, struct sigevent *evp, timer_t *timerid)
 {
 	struct timer_obj *timer;
+	const k_timeout_t ALLOC_TIMEOUT = K_MSEC(CONFIG_TIMER_CREATE_WAIT);
```

Comment:
`ALLOC_TIMEOUT` should be lowercase. Otherwise, LGTM and thanks for the PR :-)

Reply:
I am fine to make this change but I thought most conventions recommended constant variables to be upper case.  I just want to double check before I make the change.

Reply:
@nslowell typically only macros / enums are uppercase.

```bash
grep -R "const .* [A-Z0-9_]\+ =" * | wc -l
      17
cfriedt@Christophers-MacBook-Air zephyr % grep -R "const .* [a-z0-9_]\+ =" * | wc -l
    3770
```

Reply:
Thanks!